src/commands/mobile_app/upload.rs

```diff
@@ -134,13 +135,37 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
                     .map(Cow::Owned)
             });
 
-        (vcs_provider, head_repo_name)
+        let head_ref: Option<Cow<'_, str>> = matches
```

Comment:
Type annotation unnecessary here. 
```suggestion
        let head_ref = matches
```

Reply:
I have noticed Cursor tends to add unnecessary type annotations quite frequently, and sadly it is not possible to lint against them when the type involves generics (as it does here), so I have tried adding a Cursor rule in https://github.com/getsentry/sentry-cli/pull/2717. 

Hopefully that helps going forward!

Reply:
Ha, funny enough this was manual. Removed!